integration

```diff
@@ -110,6 +110,7 @@
   "aronkahrs-us/inumet-weather-ha",
   "aropop/home-assistant-cronicle",
   "artspb/homeassistant-tk-husteblume",
+  "asachs01/ha-epb",
```

Comment:
As this is only relevant for users in a specific country, the released `hacs.json` file needs a country key.
<https://hacs.xyz/docs/publish/include>